spec/lucky_web/action_redirect_spec.cr

```diff
@@ -15,7 +15,7 @@ describe LuckyWeb::Action do
     should_redirect(action, to: "/somewhere", status: 302)
 
     action = RedirectAction.new(build_context, params)
-    action.redirect to: RedirectAction.route
+    action.redirect to: RedirectAction.with
```

Comment:
Even though there is no built-in `alias_method` in Crystal we could do something like this: https://play.crystal-lang.org/#/r/3076. Because you're right that `with` without params is pretty weird